Samples/iOS-Swift/iOS-Swift-UITests/UserFeedbackUITests.swift

```diff
@@ -82,7 +80,9 @@ class UserFeedbackUITests: BaseUITest {
         
         app.staticTexts["Send Bug Report"].tap()
         
-        // TODO: need to implement some kind of error dialog and then assert for it
+        XCTAssert(app.staticTexts["Error"].exists)
```

Comment:
`m`: I think it would be great to validate the actual error message in the tests.

Reply:
Disagree; this is set with standard API, and we are already validating that the alert is showing. Testing that kind of detail is testing the Apple framework, not anything we have control over. https://github.com/getsentry/sentry-cocoa/blob/e61b1088ff17cfa6432913f43115647a0cab83eb/Sources/Swift/Integrations/UserFeedback/SentryUserFeedbackForm.swift#L124-L126

Reply:
Then all good

Reply:
Actually I reconsidered this and added many more assertions for various things like this in https://github.com/getsentry/sentry-cocoa/pull/4600